packages/core/upload/admin/src/pages/App/tests/MediaLibrary.test.js

```diff
@@ -224,15 +134,6 @@ describe('Media library homepage', () => {
       expect(screen.getByRole('main').getAttribute('aria-busy')).toBe('true');
       expect(screen.getByText('Loading content.')).toBeInTheDocument();
     });
-
-    it('shows a loader while resolving the folder structure', () => {
```

Comment:
I removed it because the media library doesn't directly load the folder structure anymore (it is now part of the header component you have created) and therefore the loading state is not dependent on it any longer.

Reply:
> can we mockReturnValueOnce with your new mocks?

Yes, you can. The mock returns a `jest.fn()` function, which can be used just we way you are used to it :) An example: https://github.com/strapi/strapi/blob/9b7c0ca077b296db71f221d0850fcf2c4ea872ef/packages/core/upload/admin/src/components/AssetDialog/tests/AssetDialog.test.js#L61

Reply:
nice, thanks!